doc/source/reference/apis/metadata.md

```diff
@@ -159,54 +183,50 @@ Example response:
   "name": "my-model",
   "versions": ["my-model/v1"],
   "platform": "platform-name",
-  "inputs": [{"datatype": "BYTES", "name": "input", "shape": [1, 5]}],
-  "outputs": [{"datatype": "BYTES", "name": "output", "shape": [1, 3]}],
+  "inputs": [{"messagetype": "tensor", "schema": {"shape": [1, 5]}}],
+  "outputs": [{"messagetype": "tensor", "schema": {"shape": [1, 3]}}]
 }
 ```
 
 
-## V1/V2 format reference
+## SeldonMessage metadata vs kfserving TensorMetadata
```

Comment:
I think we can change vs, instead something like "More details on seldonmessage and kfserving metadata"